streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/utils/StreamToTableJoinTestData.scala

```diff
@@ -1,10 +1,12 @@
 /*
- * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
- * Copyright (C) 2017-2018 Alexis Seigneurin.
```

Comment:
I'm not sure of the implications of adding these copyright lines, or whether it's proper to remove the prior lines. Maybe there's an armchair OS lawyer out there who can comment.

Reply:
Humm that one was not intended, I will revert that.